robot.py

```diff
@@ -48,6 +48,8 @@ class Robot(magicbot.MagicRobot):
     back_lift: Lift
     lift_drive: LiftDrive
 
+    vision: Vision
```

Comment:
I think this is just acting as a type hint right now - a Vision instance is being created in createObjects.